server/server.go

```diff
@@ -410,8 +410,10 @@ func validateOptions(o *Options) error {
 	if err := validateLeafNode(o); err != nil {
 		return err
 	}
-	// Check that gateway is properly configured. Returns no error
```

Comment:
Did we lose this comment?

Reply:
The gateway one.

Reply:
@derekcollison Good find. Mind if I add it back in with the change that calls validateOptions after reload?

Reply:
sure.